Analysis/Utkscan/core/source/UtkUnpacker.cpp

```diff
@@ -60,6 +60,9 @@ void UtkUnpacker::ProcessRawEvent(ScanInterface *addr_/*=NULL*/) {
         PrintProcessingTimeInformation(systemStartTime, times(&systemTimes),
             GetEventStartTime(), eventCounter);
 
+    //set eventWidth from Globals (loaded from config file)
+    eventWidth = Globals::get()->GetEventLengthInTicks();
```

Comment:
This line should use the Unpacker::SetEventWidth() method, eventWidth is protected.